docs/guides/remote-builder.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Remote builder"
+description: "Connect buildx to an external buildkitd instance"
+keywords: build, buildx, buildkit
+---
+
+The buildx remote driver allows for more complex custom build workloads that
+allow users to connect to external buildkit instances. This is useful for
+scenarios that require manual management of the buildkit daemon, or where a
+buildkit daemon is exposed from another source.
+
+To connect to a running buildkitd instance:
+
+```console
+$ docker buildx create \
+  --name remote \
+  --driver remote \
+  tcp://localhost:1234
+```
+
+## Remote Buildkit over Unix sockets
+
+In this scenario, we'll create a setup with buildkitd listening on a unix
+socket, and have buildx connect through it.
+
+Firstly, ensure that [buildkit](https://github.com/moby/buildkit) is installed.
+Then launch an instance of buildkitd:
+
+```console
+$ sudo ./buildkitd --group $(id -gn) --addr unix://$HOME/buildkitd.sock
+```
```

Comment:
I’d rather recommend using rootless mode, but that can be another PR

Reply:
Lets keep it simple as it is and add a link to https://github.com/moby/buildkit/blob/master/docs/rootless.md for rootless. Could even leave the default socket path. Can also point to https://github.com/moby/buildkit/tree/master/examples/systemd for running buildkitd with systemd.

Reply:
Have setup those links right below this to point users in the right direction - I think we should keep the specified socket path, otherwise we'll also need to tell users later on to use sudo buildx (since the `/var/run/buildkit/` directory is owned by `root:root` and inaccessible, even with `--group`, though maybe that's a bug?).